target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -111,6 +111,17 @@ define Device/tplink_archer-c6-v2-us
 endef
 TARGET_DEVICES += tplink_archer-c6-v2-us
 
+define Device/tplink_tl-wa1201-v2
+  $(Device/tplink-safeloader-uimage)
+  SOC := qca9563
+  IMAGE_SIZE := 7808k
+  DEVICE_MODEL := TL-WA1201
+  DEVICE_VARIANT := v2 (EU)
```

Comment:
Will US, RU, AU etc. be different? Should we also include the -eu in the compatible/image name?

Reply:
I have two EU devices and as far as I can tell the NA has the same hardware. I don't know about RU and AU. How should I handle this?

Reply:
Wait for other people's comments, download and compare firmware, ...

But in any case you should decide whether this a support for EU-only, or whether it aims at all TL-WA1201 devices by not specifying any country, so people looking for it know what to expect.

Reply:
EU/NA/RU/AU all have the exact same firmware files on the tp-link's website. I'll remove the EU specifier and keep this port generic.

Reply:
Okay, that seems right.